worknenjoy/gitpay
# Issue#799

## Description

> Change the status of task if is different of status from GitHub.

## Changes

- task-header.js
- taskFetch.js

## Issue

> Closes #799 

## Impacted Area

> Task Page

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create / log user
- Import Task
- Open Task to see if status is the same of GitHub and the action buttons are disabled.

## Before
> ![image](https://user-images.githubusercontent.com/21283066/115739507-5dc23d80-a364-11eb-9d89-61dbabb6b5c9.png)


## After
> ![image](https://user-images.githubusercontent.com/21283066/115743042-a92a1b00-a367-11eb-86df-dfb0e2bd57a2.png)
